src/main/java/frc/robot/Config.java

```diff
@@ -44,10 +45,18 @@ public final class Config {
    */
   private static int robotId = -1;
   public static class CANID {
+    public static int PIGEON = robotSpecific(30, 27, 27, 27, 30);
```

Comment:
could put it into class DIFF.
class Swerve has its own pigeonID.

Reply:
Sorry I disagree with Wei's comment. The PIGEON canid should belong where you put it, which is in the CANID nested class.

Reply:
I think you can keep this `PIGEON` constant here, I will ask the swerve people to delete their constant and use yours once your code is in main.

Reply:
I agree to leave PIGEON canid here. And remove the one in Swerve class.
Since we only have one PIGEON on the robot.